src/chrome/content/toolbar_button.xul

```diff
@@ -43,6 +43,8 @@
       <menupopup id="https-everywhere-context" onpopupshowing="show_applicable_list(this)">
         <!-- entries will be written here by ApplicableList.populate_menu() -->
         <menuseparator class="hide-on-disable"/>
+        <menuitem label="&https-everywhere.prefs.reset_defaults;"
```

Comment:
Logically, it seems this option should be in the same menu subsection as the ruleset togglers...